.github/workflows/build.yml

```diff
@@ -21,11 +32,17 @@ jobs:
     - name: Build project
       run: dotnet build -bl:build.binlog --configuration Release
 
+    - name: Get Path to Tests
+      run: echo "TEST_PATH=$(dotnet msbuild SIL.Core.Tests/ -p:TargetFramework=${{ matrix.framework }} --getProperty:OutputPath -p:Configuration=Release)" >> $GITHUB_ENV
+
+    - name: Print
+      run: echo "$TEST_PATH"
+
     # there are cases where this will fail and we want to know about it
     # so we don't use continue-on-error, but we still want to publish the results
     - name: Test project
       id: test
-      run: dotnet test -bl:test.binlog --filter "TestCategory != SkipOnTeamCity" --blame-hang-timeout 5m --logger:"trx;LogFilePrefix=results" --results-directory ./test-results
+      run: dotnet test "$TEST_PATH"/SIL*Tests.dll --filter "TestCategory != SkipOnTeamCity" --blame-hang-timeout 5m --logger:"trx;LogFilePrefix=results" --results-directory ./test-results
```

Comment:
Instead of `$TEST_PATH` you want `${{ env.TEST_PATH }}` the same syntax as the concurrency value above

Reply:
Interesting. Why is that? That variable appears to be working